packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts

```diff
@@ -1146,3 +1149,54 @@ test('grants createInvalidation', () => {
     },
   });
 });
+
+test('render distribution behavior with realtime log config', () => {
+  const role = new iam.Role(stack, 'Role', {
+    assumedBy: new iam.ServicePrincipal('cloudfront.amazonaws.com'),
+  });
+
+  const stream = new kinesis.Stream(stack, 'stream', {
+    streamMode: kinesis.StreamMode.ON_DEMAND,
+    encryption: kinesis.StreamEncryption.MANAGED,
+  });
+
+  const realTimeConfig = new RealtimeLogConfig(stack, 'RealtimeConfig', {
+    endPoints: [
+      Endpoint.fromKinesisStream(stream, role),
+    ],
+    fields: ['timestamp'],
+    name: 'realtime-config',
+    samplingRate: 50,
+  });
+
+  new Distribution(stack, 'MyDist', {
+    defaultBehavior: {
+      origin: defaultOrigin(),
+      realtimeLogConfig: realTimeConfig,
+    },
+  });
+
+  Template.fromStack(stack).hasResourceProperties('AWS::CloudFront::Distribution', {
+    DistributionConfig: {
```

Comment:
We can use `Match.objectLike` here to only check the relevant part of the test (`RealtimeLogConfigArn`)

Reply:
Fixed 👍